src/services/auth.service.ts

```diff
@@ -0,0 +1,28 @@
+import {Users} from '../loaders/database';
+import bcrypt from 'bcrypt';
+import {AUTH_MESSAGES} from '../config/messages';
+
+export default {
+  async login(login: string, password: string) {
```

Comment:
I think you can create JWT token here and remove this logic from controller layer

Reply:
fixed